projects/riverbankcomputing.com/pyqt-builder/package.yml

```diff
@@ -0,0 +1,25 @@
+distributable:
+  url: https://www.riverbankcomputing.com/hg/PyQt-builder/archive/{{version}}.tar.gz
+  strip-components: 1
+versions:
+  url: https://www.riverbankcomputing.com/hg/PyQt-builder/tags
+  match: /rev\/\d+\.\d+\.\d+/
+  strip:
+    - /^rev\//
+runtime:
+  env:
+    PYTHONPATH: "{{prefix}}/venv/lib/python{{deps.python.org.version.marketing}}/site-packages:$PYTHONPATH"
```

Comment:
I think this has the same issue as pycairo. no runtime dep to generate that from.